README.md

```diff
@@ -135,6 +135,24 @@ sudo usermod -a -G libvirt $USER
 </p>
 </details>
 
+<details><summary>Void</summary>
```

Comment:
May I suggest to call the section `Void Linux`, instead of just `Void`. Void alone is a bit confusion.